node-operators/networks/run-a-node/docker.md

```diff
@@ -0,0 +1,220 @@
+
+![Full Node Starting](/images/node-operators/networks/run-a-node/docker/full-node-docker-1.png)
+
+!!! note
+    If you want to run an RPC endpoint, to connect polkadot.js.org, or to run your own application, use the flags `--unsafe-rpc-external` and/or `--unsafe-ws-external` to run the full node with external access to the RPC ports.  More details are available by running `moonbeam --help`.  
+
+!!! note
+    You can specify a custom Prometheus port with the `--prometheus-port XXXX` flag (replacing `XXXX` with the actual port number). This is possible for both the parachain and embedded relay chain.
+
+
+```
+docker run -p {{ networks.relay_chain.p2p }}:{{ networks.relay_chain.p2p }} -p {{ networks.parachain.p2p }}:{{ networks.parachain.p2p }} -p {{ networks.parachain.rpc }}:{{ networks.parachain.rpc }} -p {{ networks.parachain.ws }}:{{ networks.parachain.ws }} #rest of code goes here
+```
+
+During the syncing process, you will see messages from both the embedded relay chain and the parachain (without a tag). These messages display a target block (live network state) and a best block (local node synced state).
+
+![Full Node Starting](/images/node-operators/networks/run-a-node/docker/full-node-docker-2.png)
+
+!!! note
+    It will take a few days to completely sync the embedded Kusama relay chain. Make sure that your system meets the [requirements](/node-operators/networks/run-a-node/overview#requirements). 
+
+If you followed the installation instructions for Moonbase Alpha, once synced, you will have a node of the Moonbase Alpha TestNet running locally!
+
+If you followed the installation instructions for Moonriver, once synced, you will be connected to peers and see blocks being produced on the Moonriver network! Note that in this case you need to also sync to the Kusama relay chain, which might take a few days.
+
+## Updating the Client {: #updating-the-client } 
+
+As Moonbeam development continues, it will sometimes be necessary to upgrade your node software. Node operators will be notified on our [Discord channel](https://discord.gg/PfpUATX) when upgrades are available and whether they are necessary (some client upgrades are optional). The upgrade process is straightforward and is the same for a full node or collator.
+
+First, stop the docker container:
+
+```
+sudo docker stop `CONTAINER_ID`
+```
+
+Then, install the new version by repeating the steps described before, making sure that you are using the latest tag available. After updating, you can start the service again.
+
+### Purging the Chain {: #purging-the-chain } 
+
+On an as-needed basis, Moonbase Alpha might be purged and reset. If a purge is required, node operators will be notified in advance (via our [Discord channel](https://discord.gg/PfpUATX)). You can also purge your node if your individual data directory becomes corrupted.
```

Comment:
Remove on an as-needed basis, make this section more about general purging versus a testnet reset